tests/puppeteer/pages/BO/catalog/stocks/index.js

```diff
@@ -92,8 +98,45 @@ module.exports = class Stocks extends BOBasePage {
         return this.getTextContent(this.productRowReferenceColumn.replace('%ROW', row));
       case 'supplier':
         return this.getTextContent(this.productRowSupplierColumn.replace('%ROW', row));
+      case 'physical':
+        return this.getNumberFromText(this.productRowPhysicalColumn.replace('%ROW', row));
+      case 'reserved':
+        return this.getNumberFromText(this.productRowReservedColumn.replace('%ROW', row));
+      case 'available':
+        return this.getNumberFromText(this.productRowAvailableColumn.replace('%ROW', row));
       default:
         throw new Error(`${column} was not find as column in this table`);
```

Comment:
Do we have to take into account this kind of mistake ?

Reply:
this will make test fail at the right place and not with the right reason